docs/developer/data_address_types.md

```diff
@@ -0,0 +1,124 @@
+# A Note On Data Address Types
+
+A **_Data Address_** is a location reference used by EDC runtimes. This note outlines the different data address types and their use. 
+
+## Control Plane
+
+There are three types of data addresses in the control plane: Destination Data Addresses (DDA); Asset Data Addresses (ADA); and Content Data Addresses (CDA). 
+
+#### Destination Data Address (DDA)
+
+A consumer EDC creates a **_Destination Data Address_** for provider push transfers. The DDA is generated by the consumer and sent to the provider via a data transfer request. The provider uses the DDA to push data to the consumer.
+
+#### Asset Data Address (ADA)
+
+An **_Asset Data Address_** is associated with an asset on the provider and used by the latter to locate asset data. 
+
+#### Content Data Address (CDA)
+
+A **Content Data Address_** is used to resolve asset data. An CDA may be used by the provider to resolve data that is in turn pushed to the consumer as specified by a DDA. Alternatively, in pull transfer scenarios, an CDA may be sent to the consumer, which uses it to access the data.
+
+#### ADAs and SDAs in the Control Plane
+
+In some cases, an ADA may be used as an CDA; in other cases, there is no direct mapping. Consider two scenarios.
+
+##### Scenario 1: Client API Access
+
+A consumer requests API access modeled as an asset from a provider. The provider resolves the ADA and passes it back to the consumer to pull the data (access the API). In this case, the ADA and CDA are the same.
```

Comment:
This case can be more complex as the provide control plane can actually provide a CDA which is not the ADA. For example the CDA can be the public API of the Data Plane, which is used as proxy to access the ADA (the ADA is wrapped in the access token passed to the consumer for accessing the public API of the DPF)